lib/settings_base.py

```diff
@@ -1250,8 +1250,6 @@ def read_only_mode(env):
 BUILDER_SECRET_KEY = 'love will tear us apart'
 # The builder URL we hit to upgrade jetpacks.
 BUILDER_UPGRADE_URL = 'https://addons.mozilla.org/services/builder'
```

Comment:
I believe we can get completely rid of all those `BUILDER_*` settings, right? @kmaglione?